### PR TITLE
feat: added first draft for curve distance plot.

### PR DESCRIPTION
**Pull request by Schnjan**

### CODE REVIEW

File: src/Formplot/FileFormat/CurveDistanceGeometry.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+{
+	/// <summary>
+	/// Representation of a curve distance geometry.
+	/// </summary>
+	public class CurveDistanceGeometry : Geometry
+	{
+		#region properties
+
+		/// <inheritdoc />
+		public override GeometryTypes GeometryType { get; } = GeometryTypes.CurveDistance;
```
Comment (georg-eckert-zeiss) [inline src/Formplot/FileFormat/CurveDistanceGeometry.cs:21]: Use `=>`?

File: src/Formplot/FileFormat/FormplotConverter.cs [hunk 2]
```diff
@@ -75,6 +87,24 @@ public static bool TryConvert<TPlot>( Formplot source, out TPlot? target )
 			}
 		}
 
+		private static CurveProfilePlot? ConvertToCurveProfile( Formplot source )
+		{
+			switch( source )
+			{
```
Comment (georg-eckert-zeiss) [inline src/Formplot/FileFormat/FormplotConverter.cs:93]: Switch expression?

File: src/Formplot/Formplot.csproj [hunk 3]
```diff
@@ -9,7 +9,7 @@
     <Copyright>Copyright © $([System.DateTime]::UtcNow.Year) Carl Zeiss Industrielle Messtechnik GmbH</Copyright>
     <Company>Carl Zeiss Industrielle Messtechnik GmbH</Company>
     <Authors>$(Company)</Authors>
-    <PackageVersion>3.0.7</PackageVersion>
+    <PackageVersion>3.1.0</PackageVersion>
```
Comment (georg-eckert-zeiss) [inline src/Formplot/Formplot.csproj:12]: Release pipeline will change that. Please remove it.

**Review by georg-eckert-zeiss — CHANGES_REQUESTED** [review]
Please remove version change. Other notes are recommendations.

File: src/Formplot/FileFormat/CurveDistanceGeometry.cs (hunk 1 above)
Comment (Schnjan) [reply src/Formplot/FileFormat/CurveDistanceGeometry.cs]: Done

File: src/Formplot/Formplot.csproj (hunk 3 above)
Comment (Schnjan) [reply src/Formplot/Formplot.csproj]: Done

**Review by georg-eckert-zeiss — APPROVED** [review]

**Pull request merged**